zipfile2/_zipfile.py

```diff
@@ -222,17 +221,17 @@ def _extract_member_to(self, member, arcname, targetpath, pwd,
             finally:
                 source.close()
 
-        if preserve_permissions in (PERMS_PRESERVE_SAFE, PERMS_PRESERVE_ALL):
-            if preserve_permissions == PERMS_PRESERVE_ALL:
-                #preserve bits 0-11: sugrwxrwxrwx, this include
-                #sticky bit, uid bit, gid bit
-                mode = member.external_attr >> 16 & 0xFFF
-            elif PERMS_PRESERVE_SAFE:
-                #preserve bits 0-8 only: rwxrwxrwx
-                mode = member.external_attr >> 16 & 0x1FF
-            os.chmod(targetpath, mode)
-
-        return targetpath
+            if preserve_permissions in (PERMS_PRESERVE_SAFE, PERMS_PRESERVE_ALL):
+                if preserve_permissions == PERMS_PRESERVE_ALL:
+                    #preserve bits 0-11: sugrwxrwxrwx, this include
+                    #sticky bit, uid bit, gid bit
+                    mode = member.external_attr >> 16 & 0xFFF
+                elif PERMS_PRESERVE_SAFE:
+                    #preserve bits 0-8 only: rwxrwxrwx
+                    mode = member.external_attr >> 16 & 0x1FF
+                os.chmod(targetpath, mode)
+
+            return targetpath
```

Comment:
I was referring to `this` line given  https://github.com/cournape/zipfile2/pull/19/files#r34140132 which was fixed in the meantime.